readme.md

```diff
@@ -14,6 +14,8 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
 
 ## Quick links
 
+1. [find a stale file](#findFile)
```

Comment:
nit: I notice all IDs below are in all lowercase. Could you change these to all lowercase? Also is there a reason these two items are on the top? I'd believe things like "copy a file" is more common?